### PR TITLE
Version bump for release 2.2.0

### PR DESCRIPTION
**Pull request by nishant8887**
## Context

Version bump for new release.

### Changes

Required for PR:
https://github.com/transferwise/pipelinewise-transform-field/pull/28

## Checklist
- [ ] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [ ] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

**Comment by nishant8887** [comment]
@Samira-El It would be nice if you could merge this PR and release the new version.
I need to make changes to the pipelinewise main repository. Have already raised a PR but it's still WIP.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 2.2.0 (2021-09-15)
```
Comment (Samira-El) [inline CHANGELOG.md:3]:
```suggestion
## 2.2.0 (2021-09-17)
```

**Review by Samira-El — CHANGES_REQUESTED** [review]

**Comment by Samira-El** [comment]
can you rebase the branch please? it's outta date with master

**Comment by nishant8887** [comment]
@Samira-El Rebased the branch with current master. 👍

**Review by Samira-El — APPROVED** [review]

**Comment by nishant8887** [comment]
@Samira-El Thanks 👍

**Comment by Samira-El** [comment]
https://pypi.org/project/pipelinewise-transform-field/2.2.0/

**Pull request merged**